server/RemoteHubClient.ts

```diff
@@ -190,4 +199,29 @@ export default class RemoteHubClient extends BaseRemoteClient {
       else callback(null);
     });
   };
+
+  private onImportProject =  (data: ImportProjectData) => {
+
+  fs.writeFile(path.join(this.server.projectsPath, "project.zip"), data.data as string, (err) => {
```

Comment:
A bit dangerous to write the ZIP file directly inside the project folder, since it could be overwritten by another import at the same time. If saving it to disk is actually needed (can't we unzip from memory?), would be best to save it in the system's temporary folder with a random name. We might already do that elsewhere, look for yauzl usage I guess.